installer/packer/scripts/kov/configure_kov.sh

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/bash
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -euf -o pipefail
+
+umask 077
+
+deploy=$(ovfenv -k kubernetes_manager.deploy)
+port=$(ovfenv -k kubernetes_manager.port)
+
+if [ ${deploy,,} != "true" ]; then
+  echo "Not configuring KOV and disabling startup"
+  systemctl stop kov 
```

Comment:
Should this be `systemctl disable kov --now` to prevent it from running at startup?